src/m365/spo/commands/tenant/tenant-site-membership-list.ts

```diff
@@ -0,0 +1,166 @@
+import GlobalOptions from '../../../../GlobalOptions.js';
+import { Logger } from '../../../../cli/Logger.js';
+import { odata } from '../../../../utils/odata.js';
+import { spo } from '../../../../utils/spo.js';
+import { validation } from '../../../../utils/validation.js';
+import SpoCommand from '../../../base/SpoCommand.js';
+import commands from '../../commands.js';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  siteUrl: string;
+  role?: string;
+}
+
+interface IMembershipResult {
+  userGroup: IUserInfo[];
+}
+
+interface IMembershipOutput {
+  AssociatedOwnerGroup?: IUserInfo[];
+  AssociatedMemberGroup?: IUserInfo[];
+  AssociatedVisitorGroup?: IUserInfo[];
+}
+
+interface IUserInfo {
+  email: string;
+  loginName: string;
+  name: string;
+  userPrincipalName: string;
+  associatedGroupType?: string;
+}
+
+class SpoTenantSiteMembershipListCommand extends SpoCommand {
+  public static readonly RoleNames: string[] = ['Owner', 'Member', 'Visitor'];
+
+  public get name(): string {
+    return commands.TENANT_SITE_MEMBERSHIP_LIST;
+  }
+
+  public get description(): string {
+    return `Retrieve information about default site groups' membership`;
```

Comment:
Let's make the same edit in the docs.

```suggestion
    return `Retrieves information about default site groups' membership`;
```

Reply:
@milanholemans, Ok I have changed these last two things. Thanks once again!